packages/ua-devtools/src/oapp/config.ts

```diff
@@ -240,84 +240,126 @@ export const configureReceiveLibraryTimeouts: OAppConfigurator = withOAppLogger(
+                )
 
-            if (!hasUlnConfig) {
-                const newSetConfigs: SetConfigParam[] = await endpointSdk.getUlnConfigParams(currentSendLibrary, [
-                    { eid: to.eid, ulnConfig: config.sendConfig.ulnConfig },
-                ])
-
-                // Updates map with new configs for that OApp and Send Library
-                const setConfigsByLibrary = setConfigsByEndpointAndLibrary.getOrElse(from, () => new Map())
-                const existingSetConfigs = setConfigsByLibrary.get(currentSendLibrary) ?? []
-                setConfigsByEndpointAndLibrary.set(
-                    from,
-                    setConfigsByLibrary.set(currentSendLibrary, [...existingSetConfigs, ...newSetConfigs])
+                logger.verbose(
+                    `Checked Uln configuration for ${formatOmniVector({ from, to })}: ${printBoolean(hasUlnConfig)}`
                 )
+
+                if (!hasUlnConfig) {
+                    const newSetConfigs: SetConfigParam[] = await endpointSdk.getUlnConfigParams(currentSendLibrary, [
+                        { eid: to.eid, ulnConfig: config.sendConfig.ulnConfig },
+                    ])
+
+                    // Updates map with new configs for that OApp and Send Library
+                    const setConfigsByLibrary = setConfigsByEndpointAndLibrary.getOrElse(from, () => new Map())
+                    const existingSetConfigs = setConfigsByLibrary.get(currentSendLibrary) ?? []
+                    setConfigsByEndpointAndLibrary.set(
+                        from,
+                        setConfigsByLibrary.set(currentSendLibrary, [...existingSetConfigs, ...newSetConfigs])
+                    )
+
+                    const updatedConfigList =
+                        setConfigsByEndpointAndLibrary.getOrElse(from, () => new Map()).get(currentSendLibrary) ?? []
+                    const updatedConfigListCsv = updatedConfigList
+                        .map(
+                            (setConfigParam) =>
+                                '{configType: ' + setConfigParam.configType + ', config: ' + setConfigParam.config + '}'
+                        )
+                        .join(', ')
+                    logger.verbose(
+                        `Set Uln configuration ${updatedConfigListCsv} for ${formatOmniVector({ from, to })}`
+                    )
+                }
             }
         }
-    }
 
-    // This function iterates over the map (OApp -> SendLibrary -> SetConfigParam[]) to execute setConfig
-    return buildOmniTransactions(setConfigsByEndpointAndLibrary, createSdk)
-}
+        // This function iterates over the map (OApp -> SendLibrary -> SetConfigParam[]) to execute setConfig
+        return buildOmniTransactions(setConfigsByEndpointAndLibrary, createSdk)
+    },
+    {
+        onStart: (logger) => logger.verbose(`Checking send configuration`),
+        onSuccess: (logger) => logger.verbose(`${printBoolean(true)} Checked send configuration`),
```

Comment:
change 5

Reply:
Noticed that when set to `verbose` this log doesn't appear in the test build run. When set to `info`, it appears.

Other logs within the actual logic appear, even with the verbose flag, so I can leave those as is, but should I change these `onStart` and `onSuccess` logs to use `info`?